scripts/generate_and_eval_single_sample.py

```diff
@@ -57,6 +57,8 @@ def __init__(self):
         self.log_generated_kernel = False
         self.log_eval_result = False
 
+        self.framework = "cuda" # cuda or triton
```

Comment:
nit: make this an enum instead, will make it easier to scale to more frameworks later